webook/arrangement/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class UserHasNoPersonException(Exception):
+    pass
```

Comment:
Did i see this in another PR?

Reply:
Yeah, branched from #219. Maybe we should discuss if branching from an active PR, and making changes on it, is a sane idea? I wanted to split this up into two distinct PRs as there was a sort of logical split between them.